app/resources/locale/en/en.po

```diff
@@ -1107,134 +1098,132 @@ msgid ""
 "This release has been possible thanks to the great work \n"
 "              done by an emerging team from"
 msgstr ""
+"This release has been possible thanks to the great work \n"
+"              done by an emerging team from"
 
-#: app/scripts/services/graph.js:521
+#: app/scripts/services/graph.js:532
 msgid ""
 "To enter on \"edit mode\" of deeper block, you need to finish current \"edit "
 "mode\", lock the keylock to do it."
 msgstr ""
 "To enter on \"edit mode\" of deeper block, you need to finish current \"edit "
 "mode\", lock the keylock to do so."
```

Comment:
Globally search 'keylock' -> 'padlock' - Note here the msgid has diverged from msgstr

My change suggestion:
"To enter \"edit mode\" of deeper block, you need to lock the padlock to finish \"edit "
"mode\" of current block."